src/components/tasks/card/card.module.scss

```diff
@@ -5,6 +5,22 @@ $yellow: #ecef08;
 $white: #e5e5e5;
```

Comment:
How is this white :O

Reply:
it's light shade of grey

Reply:
scss color variables refactor can be done with new PR